File: projects/mi-program/005-loops/python/loops.py
#!/usr/bin/env python3
import sys

arg = sys.argv[1] if len(sys.argv) > 1 else None

while True:
    text = arg if arg is not None else input("Enter a number (0-100000): ")
    if text.isdigit():
        n = int(text)
        if 0 <= n <= 100000:
            for i in range(1, 11):
                print(f"{i} x {n} = {i * n}")
            break
    print("Invalid input. Please enter an integer between 0 and 100000.")
    arg = None

File: projects/mi-program/005-loops/php/loops.php
#!/usr/bin/env php
<?php
$arg = $argv[1] ?? null;

while (true) {
    $text = $arg !== null ? $arg : trim(fgets(STDIN));
    if (ctype_digit($text) && $text >= 0 && $text <= 100000) {
        $n = (int)$text;
        for ($i = 1; $i <= 10; $i++) {
            echo "$i x $n = " . ($i * $n) . "\n";
        }
        break;
    }
    echo "Invalid input. Please enter an integer between 0 and 100000.\n";
    $arg = null;
}
